CHANGELOG.md

```diff
@@ -19,21 +19,18 @@ The REST API versions in the table are just for your information as the values a
 
 ### Added
```

Comment:
-   Insight export into CSV or XLSX ([doc](https://sdk.gooddata.com/gooddata-ui/docs/on_export_ready.html))

-   Support for attribute cell grouping in pivot tables when a table is sorted by first attribute ([doc](https://sdk.gooddata.com/gooddata-ui/docs/pivot_table_component.html#grouping))

-   Turning on/off totals in pivot tables via the totals menu ([doc](https://sdk.gooddata.com/gooddata-ui/docs/pivot_table_component.html#totals))

-   Specifying attribute filters by an attribute element value (see `textFilter` in [Filter Visual Components](https://sdk.gooddata.com/gooddata-ui/docs/filter_visual_components.html) and [Set Up an AFM Query](https://sdk.gooddata.com/gooddata-ui/docs/afm.html))

-   Optional stacking ([doc](https://sdk.gooddata.com/gooddata-ui/docs/chart_config.html#configure-stacking))